pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -299,21 +300,26 @@ def heartbeat(self):
         if self._rpc is not None and self._rpc.is_active:
             self._rpc.send(types.StreamingPullRequest())
 
-    def open(self, callback):
+    def open(self, callback, on_callback_error):
```

Comment:
FWIW, this method signature change is safe, as `StreamingPullManager` class is only used internally in the subscriber client's `subscribe()` method, and is not mentioned anywhere in the [user docs](https://googleapis.github.io/google-cloud-python/latest/pubsub/), at least from what I can see.